go/client/util.go

```diff
@@ -4,14 +4,76 @@
 package client
 
 import (
+	"golang.org/x/net/context"
```

Comment:
can you rename this file away from `util.go` and to something that indicates that it's mainly concerned with passphrase?